### PR TITLE
Unity Catalog Export: Documentation

### LINKED ISSUE #6966: Unity catalog table registration hook
Using the output of the Delta Lake catalog exporter (#6965), configure an external table in Databricks' Unity catalog, where the schema is the branch name.

A minimal design can be found [here](https://github.com/treeverse/lakeFS/tree/master/design/open/unity-catalog-exporter.md).

A part of https://github.com/treeverse/lakeFS/issues/6771

### PR DESCRIPTION
**Pull request by Jonathan-Rosenberg**
Closes #6966

This PR includes the documentation for Unity Catalog exports.
You're more than welcome to follow along the Unity Catalog guide and see if it works as expected.

### CODE REVIEW

**Review by johnnyaug — APPROVED** [review]

File: docs/integrations/unity_catalog.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+Let's define the table descriptor and upload it to lakeFS:
+
+Save the following as `famous-people-td.yaml`:
+
+```yaml
+---
+name: famous_people
+type: delta
+catalog: my-repo-name
```
Comment (johnnyaug) [inline docs/integrations/unity_catalog.md:63]:
```suggestion
catalog: my-catalog-name
```

**Review by johnnyaug — APPROVED** [review]
Great guide!

File: docs/integrations/unity_catalog.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+access it through the unified catalog. 
+The subsequent step-by-step guide will lead you through the process of configuring a [Lua hook]({% link howto/hooks/lua.md %})
+that exports Delta Lake tables from lakeFS, and subsequently registers them in Unity Catalog.
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+1. Access to Unity Catalog
+2. An active lakeFS installation with S3 as the backing storage, and a repository in this installation.
```
Comment (talSofer) [inline docs/integrations/unity_catalog.md:26]: Let's document this also as a limitation. With our current support, exports won't work for Azure.

**Review by talSofer — APPROVED** [review]
LGTM!

File: docs/howto/catalog_exports.md [hunk 3]
```diff
@@ -78,7 +78,13 @@ Exporters are code packages accessible through [Lua integration]({% link howto/h
...
 - Symlink Exporter: Writes metadata for the table using Hive's [SymlinkTextInputFormat](https://svn.apache.org/repos/infra/websites/production/hive/content/javadocs/r2.1.1/api/org/apache/hadoop/hive/ql/io/SymlinkTextInputFormat.html)
 - AWS Glue Catalog (+ Athena) Exporter: Creates a table in Glue using Hive's format and updates the location to symlink files (reuses Symlink Exporter).
-- See a step by step guide on how to integrate with [Glue Exporter]({% link integrations/glue_metastore.md %})
+  - See a step-by-step guide on how to integrate with [Glue Exporter]({% link integrations/glue_metastore.md %})
+- Delta Lake table exporter.
+- Unity Catalog exporter: The Unity Catalog exporter serves the purpose of registering a Delta Lake table in Unity Catalog. 
+It operates in conjunction with the Delta Lake exporter. In this workflow, the Delta Lake exporter is utilized to export 
+a Delta Lake table from lakeFS. Subsequently, the obtained result is passed to the Unity Catalog exporter to facilitate 
+its registration within Unity Catalog.
+  - See a step-by-step guide on how to integrate with [Unity Catalog Exporter]({% link integrations/unity_catalog.md %})
```
[the remark this answers is not in the document]
Comment (Jonathan-Rosenberg) [reply docs/howto/catalog_exports.md]: done

**Pull request merged**